python/paddle/fluid/executor.py

```diff
@@ -719,7 +744,7 @@ def _run(self, program, exe, feed, fetch_list, feed_var_name,
         if not use_program_cache:
             exe.run(program.desc, scope, 0, True, True, fetch_var_name)
         else:
-            exe.run_cached_prepared_ctx(ctx, scope, True, True, False)
+            exe.run_cached_prepared_ctx(ctx, scope, False, False, False)
```

Comment:
as executor.run() method allows user inputting None scope, it should be set scope to default when it is None